include/operations/StandardOperation.hpp

```diff
@@ -6,81 +6,88 @@
 #ifndef QFR_STANDARDOPERATION_H
 #define QFR_STANDARDOPERATION_H
 
-#include "DDpackage.h"
-#include "GateMatrixDefinitions.h"
 #include "Operation.hpp"
+#include "dd/GateMatrixDefinitions.hpp"
 
 namespace qc {
-    using GateMatrix = std::array<dd::ComplexValue, dd::NEDGE>;
-
-    //constexpr long double PARAMETER_TOLERANCE = dd::ComplexNumbers::TOLERANCE * 10e-2;
-    constexpr fp PARAMETER_TOLERANCE = 10e-6;
-    inline bool  fp_equals(const fp a, const fp b) { return (std::abs(a - b) < PARAMETER_TOLERANCE); }
-
-    class StandardOperation: public Operation {
+    class StandardOperation final: public Operation {
     protected:
-        static void checkInteger(fp& ld) {
+        static void checkInteger(dd::fp& ld) {
             auto nearest = std::nearbyint(ld);
             if (std::abs(ld - nearest) < PARAMETER_TOLERANCE) {
                 ld = nearest;
             }
         }
 
-        static void checkFractionPi(fp& ld) {
-            auto div     = qc::PI / ld;
+        static void checkFractionPi(dd::fp& ld) {
+            auto div     = dd::PI / ld;
```

Comment:
Maybe spell out `dd::fp div = ...`? `dd::PI` is in another package and its type is not that obvious.

Reply:
agreed 👍🏼